.github/workflows/tests.yml

```diff
@@ -111,6 +111,7 @@ jobs:
   windows:
     runs-on: windows-2019
```

Comment:
Available images are here: https://github.com/actions/virtual-environments#available-environments

Wondering if we should put `windows-latest` here instead of `windows-2019`